src/Sql/Sql/help/Az.Sql.md

```diff
@@ -99,6 +99,7 @@ Gets one or more Advisors for an Azure SQL Database.
 Gets the auditing settings of an Azure SQL database.
 
 ### [Get-AzSqlDatabaseAuditing](Get-AzSqlDatabaseAuditing.md)
+**Important: This cmdlet is deprecated, [Get-AzSqlDatabaseAudit](https://docs.microsoft.com/en-us/powershell/module/az.sql/get-azsqldatabaseaudit) is replacing it.**
```

Comment:
This is the only change I see.  This is the breaking change release, so  you need to remove these attributes (and this comment in help).

Reply:
This is now resolved.